app/api/validate.py

```diff
@@ -0,0 +1,122 @@
+            return validation
+        else:
+            bucketlist = BucketList.query.filter_by(name=json['name']).first()
+            if bucketlist:
+                validation.status = False
+                validation.message = "Bucketlist already exists"
+                return validation
+            else:              
+                validation.status = True
+                validation.message = "Bucketlist successfully created!"
+                return validation
+                # validation.status = True
+                # validation.message = "Bucketlist successfully created!"
+            
+    except KeyError:
+        validation.status = False
+        validation.message = "You did not include a bucketlist name."
+        return validation
+    # else:
+    #     validation.status = False
+    #     validation.message = "Bucketlist already exists"
+    #     return validation
+
+def validate_item(json):
+    validation = Validation()
+    try:
+        if not len(json['item_name']):
+            validation.status = False
+            validation.message = "The item name is too short."
+            return validation
+        else:
+            item = Item.query.filter_by(item_name=json['item_name']).first()
+            if item:
+                validation.status = False
+                validation.message = "Item already exists"
+                return validation
+            else:              
+                validation.status = True
+                validation.message = "Item successfully created!"
+                return validation
+    except KeyError:
+        validation.status = False
+        validation.message = "You did not include an Item name."
+        return validation
+            
+    #     if json.item_name:
+    #         validation.status = False
+    #         validation.message = "Invalid data."
+    #         return validation
+    #     if 'name' in json and not len(json['name']) > 0:
+    #         validation.status = False
+    #         validation.message = "The item name is too short."
+    #     elif 'done' in json and not json['done'] in ['true', 'false']:
+    #         validation.status = False
+    #         validation.message = "The item completion status should be either 'true' or 'false'."
+    #     else:
+    #         validation.status = True
+    #         validation.message = "Item successfully created!"
+    #     return validation
+    # except KeyError:
+    #     validation.status = False
+    #     validation.message = "You did not include the item name."
+    #     return validation
+
+
+def validate_limit_and_offset(limit='20', offset=100):
+    validation = Validation()
+    try:
+        limit = int(limit)
+        offset = int(offset)
+    except ValueError:
+        validation.status = False
+    if limit > offset:
+        validation.status = False
+    elif limit < 1:
+        validation.status = False
+    validation.status = True
```

Comment:
This should be in an `else` block, as is, will always make the validation `True`.